package/MDAnalysis/visualization/streamlines.py

```diff
@@ -45,7 +45,9 @@
 import multiprocessing
 
 import numpy as np
+from numpy.typing import NDArray
```

Comment:
The CI failures seem to be just because the NumPy in use is too old for this, but I think the bump to `1.20.0` minimum is imminent anyway.